srcs/juloo.keyboard2/KeyValue.java

```diff
@@ -40,6 +40,8 @@
   public static final int FLAG_ACCENT4 = (1 << 19); // Tilde
   public static final int FLAG_ACCENT5 = (1 << 20); // Cédille
   public static final int FLAG_ACCENT6 = (1 << 21); // Tréma
+  public static final int FLAG_ACCENT6 = (1 << 22); // Caron
+  public static final int FLAG_ACCENT6 = (1 << 23); // Macron
```

Comment:
These are the next free flags:
```suggestion
  public static final int FLAG_ACCENT_CARON = (1 << 26); // Caron
  public static final int FLAG_ACCENT_MACRON = (1 << 27); // Macron
```
Also, let's start writing the accent names.